src/process.c

```diff
@@ -22,7 +22,7 @@ struct list ready_list = { 0, 0 };
 void process_init() {
     // Create a dummy process with no code and no data, and load its pagetable
     // Even though it's dummy, at least kernel memory is direct mapped, so
-    // kernel code can run as usual
+    // kernel code can run as usual (specifically to enable ata interrupts)
```

Comment:
Id still like to get some clarification here. ATA interrupts weren't possible until the mouse code and related PIC fix went in, and the filesystem code we have uses polling instead of interrupts to read data.